frontend-cert/js-projects/gradebook-app/user-stories.md

```diff
@@ -0,0 +1,23 @@
+1. You should have a function named `getAverage` that takes an array of test scores as a parameter and returns the average score.
+
+1. You should have a function named `getGrade` that takes a student score as a parameter and returns a string representing a letter grade based on the score. Here are the scores and their corresponding letter grades:
+
+    | Score Range | Grade   |
+    | ----------- | ------- |
+    | `100`       | `"A++"` |
```

Comment:
I know this is how the existing project is, but is `A++` a thing? I think we should just go with `A+` - pretty minor. It's fine how it is.

Reply:
>is `A++` a thing?

I have no idea :smiley: